<?xml version="1.0" ?>
<launch>
	<node pkg="robot_joy" type="models_joy" name="models_joy" output="screen" >
		<param name="alarm_volume" value="25" type="int" />
	</node>

	<node pkg="robot_joy" type="get_points_joy" name="get_points_joy" output="screen">
		<param name="axis_linear_x" value="1" type="int" />
		<param name="axis_angular" value="3" type="int" />
		<param name="axis_cloud_pitch" value="6" type="int" />
		<param name="axis_cloud_yaw" value="7" type="int" />
		<param name="axis_fun1" value="2" type="int" />
		<param name="axis_fun0" value="5" type="int" />
		<param name="btn_cloud_zero" value="3" type="int" />
		<param name="btn_cloud_light" value="2" type="int" />
		<param name="btn_cloud_scrub" value="1" type="int" />
		<param name="btn_openfile" value="0" type="int" />
		<param name="btn_record" value="7" type="int" />
		<param name="btn_clear" value="6" type="int" />
		<param name="btn_savefile" value="8" type="int" />
		<param name="btn_action1" value="4" type="int" />
		<param name="btn_action0" value="5" type="int" />

		<param name="fun_max" value="5" type="int" />
		<param name="linear_max" value="1" type="double" />
		<param name="angular_max" value="0.5" type="double" />
		<param name="angular_percent" value="30" type="double" />

		<param name="file_path" value="/home/robot/Explosion_proof_robot/src/robot_center/param/point.yaml" type="string" />
	</node>

	<node pkg="robot_joy" type="center_joy" name="center_joy" output="screen">
		<param name="begin1" value="4" type="int" />
		<param name="begin0" value="5" type="int" />
		<param name="path1" value="6" type="int" />
		<param name="path0" value="7" type="int" />
		<param name="action" value="8" type="int" />
		<param name="toswitch" value="9" type="int" />
	</node>

	<node pkg="robot_joy" type="creat_map_joy" name="creat_map_joy" output="screen">
		<param name="axis_linear" value="1" type="int" />
		<param name="axis_angular" value="2" type="int" />
		<param name="btn_startup" value="11" type="int" />
		<param name="btn_timesave" value="4" type="int" />
		<param name="buffersave" value="3" type="int" />
		<param name="btn_nosave" value="0" type="int" />
		<param name="btn_shut" value="10" type="int" />

		<param name="linear_max" value="1" type="double" />
		<param name="angular_max" value="0.5" type="double" />
	</node>

	<!--node respawn="true" pkg="joy" type="joy_node" name="joy_node">
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.12" />
    </node-->
</launch>
